info.yml

```diff
@@ -13,35 +13,91 @@
 
 # expected fields in the configuration file for this app
 configuration:
-    name:
+
+    default_node_name:
         type: str
         description: >
-            A human readable name to display in the HDA to allow selection of this
-            set of configuration options in Houdini.
+            A default name for Alembic output nodes created in houdini.
+            Allowed characters include letters, numbers, periods, dashes, or
+            underscores.
+        default_value: sgtk_alembic
 
     work_file_template:
         type: template
         description: >
-            A reference to a template which locates a Houdini work file on disk.
-            This is used to drive the version and optionally the name of output files.
-        required_fields: [version]
-        optional_fields: [name]
+            A reference to a template which locates a Houdini work file on
+            disk.  This is used to drive the version and optionally the name of
+            output files.
+        fields: context, version, [name]
```

Comment:
What is the context field and how is it used?  Version and name make sense, but I don't see how context is used.  This is true for the settings below that also use context.